### PR TITLE
Support key base for python function

### PR DESCRIPTION
**Pull request by wolfstudy**
### Motivation

The changing base on #8523 

### Modifications

Support key_base  for python function

### CODE REVIEW

**Review by codelipenghui — APPROVED** [review]

**Comment by sijie** [comment]
@wolfstudy the change looks good. But I think we introduced a feature to support end-to-end encryption in Pulsar Function in #8432. I think we need to include this support for python functions as well. In this change, there is code generated for crypto reader. Can you create an issue to follow up to add this support?

**Comment by wolfstudy** [comment]
> @wolfstudy the change looks good. But I think we introduced a feature to support end-to-end encryption in Pulsar Function in #8432. I think we need to include this support for python functions as well. In this change, there is code generated for crypto reader. Can you create an issue to follow up to add this support?

Sure, will process it.

File: tests/integration/src/test/java/org/apache/pulsar/tests/integration/functions/PulsarFunctionsTest.java [hunk 1]
```diff
@@ -26,15 +26,7 @@
...
-import org.apache.pulsar.client.api.Consumer;
-import org.apache.pulsar.client.api.Message;
-import org.apache.pulsar.client.api.MessageId;
-import org.apache.pulsar.client.api.Producer;
-import org.apache.pulsar.client.api.PulsarClient;
-import org.apache.pulsar.client.api.Reader;
-import org.apache.pulsar.client.api.Schema;
-import org.apache.pulsar.client.api.SubscriptionInitialPosition;
-import org.apache.pulsar.client.api.SubscriptionType;
+import org.apache.pulsar.client.api.*;
```
Comment (sijie) [inline tests/integration/src/test/java/org/apache/pulsar/tests/integration/functions/PulsarFunctionsTest.java:29]: Can we avoid using wildcard import?

Comment (wolfstudy) [reply tests/integration/src/test/java/org/apache/pulsar/tests/integration/functions/PulsarFunctionsTest.java:29]: Sure, will fix it.

**Pull request merged**